guides/common/attributes-titles.adoc

```diff
@@ -2,6 +2,7 @@
 
 :AdministeringDocTitle: Administering {ProjectName}
 :AdministeringAnsibleDocTitle: Managing {Project} with Ansible
+:AdministeringNetworkDocTitle: Administering network services
```

Comment:
I think we should go for a more specific title because there are aspects of networking that are not in this new guide. I recommend either "Configuring DNS, DHCP, and TFTP" or possibly "Configuring external network services." What do you think?

Reply:
I'd say it's more like Integrating network services because I think it should also cover configuring using an external network service. Taking upstream as an example, there's Amazon Route53 integration and for sure you're not administrating that service yourself but only integrate with it. Less extreme I'd say the same thing can be true for Infoblox or PowerDNS (which another department in your org can run).

Something I haven't made up my mind about is focusing it more on provisioning. At least DHCP and TFTP are solely used when you provision. DNS can also be used when a host is renamed, but it's a bit of an edge case IMHO.

Taking a step back, you could look at making the provisioning guide aimed at users where this new guide is aimed at admins.

Reply:
This guide currently includes setting up DNS, etc, internally within the Sat environment. I am not sure that could be considered integration. Should we use this guide only for integration with external services?

Reply:
> This guide currently includes setting up DNS, etc, internally within the Sat environment. I am not sure that could be considered integration. Should we use this guide only for integration with external services?

It's very limiting. You will have setting up DNS integration in one or two guides (like Installing Server and Installing Proxy) and then some variants of DNS integration in this new guide.

What I would like to see is a guide that says "here's how you set up DNS integration and these are your options", like I already laid out in https://github.com/theforeman/foreman-documentation/issues/2957. And then do the same for DHCP.

I also think we should drop the chapters that set up all integrations at the same time because it gives the impression you must run them together, which is completely false.

Reply:
> I also think we should drop the chapters that set up all integrations at the same time because it gives the impression you must run them together, which is completely false.

@ekohl Agree. Since that is outside the scope of this PR, I will create a task to split apart those chapters.

Reply:
@Lennonka Re: title. How about "Configuring DNS, DHCP, and TFTP integration"? I recommend keeping titles fairly specific. I think "Administering" sounds odd because we don't administer external services; we configure them. What do you think?